### DISCUSSION TITLE
Feature request: METAR

### DISCUSSION THREAD

**Discussion by chump29** [Khopa/dcs_liberation#2620] [Q&A] [answered]
It would be great to have a METAR in the UI for flight/mission planning based on the upcoming weather for the turn. Maybe a popup when you click on the weather conditions so that the UI layout does not need to be altered too much?

**Comment by Starfire13** [comment, marked as answer]
This isn't a Q&A question though. If you want a feature, please file it as a feature request.

**Reply by chump29** [reply]
My apologies. I will add it properly. This can be deleted if you wish. Thanks!
